src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -496,5 +506,18 @@ protected void cleanup(JobContext context) throws IOException {
           .execute();
       }
     }
+
+    private static String formatPartitionFilter(String partitionFilter) {
```

Comment:
please provide what is the input and output this method in the javadoc

Reply:
@CuriousVini can you please provide us some guidelines on how to do this

Reply:
Since this is modifying the filter configuration, could you add what is the user provided input and how this method modifies it and what is the resultant output